packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewer.spec.js

```diff
@@ -0,0 +1,193 @@
+import { expect } from 'chai'
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import asyncStates from '@zooniverse/async-states'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { SubjectFactory } from '@test/factories'
+import mockStore from '@test/mockStore'
+
+import ImageAndTextViewerConnector from './ImageAndTextViewerConnector'
+
+describe('ImageAndTextViewer', function () {
+  function withStore (store) {
+    return function Wrapper ({ children }) {
+      return (
+        <Grommet theme={zooTheme}>
+          <Provider classifierStore={store}>
+            {children}
+          </Provider>
+        </Grommet>
+      )
+    }
+  }
+
+  describe('with loading state of error', function () {
+    const store = mockStore()
+
+    it('should render something went wrong', function () {
+      render(
+        <ImageAndTextViewerConnector
+          loadingState={asyncStates.error}
```

Comment:
loading state should be set in the store. I don’t think it’s passed in by the classifier.